plugins/cassandra/cassandrawu.cpp

```diff
@@ -3126,6 +3153,8 @@ class CCasssandraWorkUnitFactory : public CWorkUnitFactory, implements ICassandr
             {
                 int major = versionInfo->getPropInt("@major", 0);
                 int minor = versionInfo->getPropInt("@minor", 0);
+                partitions = versionInfo->getPropInt("@numPartitions", DEFAULT_PARTITIONS);
```

Comment:
They are stored in attributes column:

    {"attributes", "map<text, text>", "@major@minor@partition@", attributeMapColumnMapper},  // name is the suppression list, note trailing @